dask/dataframe/tests/test_rolling.py

```diff
@@ -159,8 +171,8 @@ def test_rolling_methods(method, args, window, center, check_less_precise):
     )
 
     # Series
-    prolling = df.a.rolling(window, center=center)
-    drolling = ddf.a.rolling(window, center=center)
+    prolling = df.a.rolling(window, center=center, min_periods=min_periods)
+    drolling = ddf.a.rolling(window, center=center, min_periods=min_periods)
```

Comment:
It seems that this test now fails once in a few runs, see eg https://github.com/dask/dask/pull/6996/checks?check_run_id=1681392897#step:5:191